contrib/scripts/protoc-swagger-gen.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+mkdir -p ./tmp-swagger-gen
+
+# Get the path of the cosmos-sdk repo from go/pkg/mod
+proto_dirs=$(find ./proto ./third_party/proto -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
+
+for dir in $proto_dirs; do
+  # generate swagger files (filter query files)
```

Comment:
Yes, that issue is what this PR resolves. I changed the description to note that detail.

Reply:
@brianosaurus what I meant is, looks like this generates swagger, instead of helping fix the linting issue - If we don’t need it to solve the linting issue, we should move the swagger-specific stuff to another PR